docs/DeveloperGuide.md

```diff
@@ -1,38 +1,208 @@
 # Developer Guide
```

Comment:
[CRITICAL]
Diagrams are not shown on DG html page. Please fix this as soon as possible.